src/WebJobs.Extensions.DurableTask/ContextImplementations/DurableEntityContext.cs

```diff
@@ -256,22 +366,92 @@ internal void SendEntityMessage(OrchestrationInstance target, string eventName,
             }
         }
 
-        internal void SendOutbox(OrchestrationContext innerContext)
+        internal void SendResponseMessage(OrchestrationInstance target, string eventName, object message, bool isException)
+        {
+            lock (this.outbox)
+            {
+                if (message is RequestMessage requestMessage)
+                {
+                    this.State.MessageSorter.LabelOutgoingMessage(requestMessage, target.InstanceId, DateTime.UtcNow, this.EntityMessageReorderWindow);
+                }
+
+                this.outbox.Add(new ResultMessage()
+                {
+                    Target = target,
+                    EventName = eventName,
+                    EventContent = message,
+                    IsError = isException,
+                });
+            }
+        }
+
+        internal void SendLockMessage(OrchestrationInstance target, string eventName, object message, bool notifyOnFailedWriteback = false)
```

Comment:
The `notifyOnFailedWriteback` parameter is not being used.

Reply:
yes, that's a leftover from copy-paste that should not be there.